source/_integrations/media_player.group.markdown

```diff
@@ -0,0 +1,50 @@
+---
+title: "Media Player Group"
+description: "Instructions for how to setup Media player groups within Home Assistant."
+ha_category:
+  - Media Player
+ha_release: 0.117
+ha_iot_class: Local Push
+ha_quality_scale: internal
+ha_domain: group
+---
+
+The group media player platform lets you combine multiple media players into one entity. All child media players of a media player group can still be used as usual, but controlling the state of the grouped media player will forward the command to each child media player.
+
+To enable this platform in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: group
+    name: TVs
```

Comment:
Only required options in the first example

Reply:
OK